quantum/nodes/not-gate/not-gate.html

```diff
@@ -0,0 +1,29 @@
+<script type="text/javascript">
+  RED.nodes.registerType("not-gate", {
+    category: "quantum",
+    color: "#33B1FF",
+    defaults: {
+      name: { value: "x" },
+    },
+    inputs: 1,
+    outputs: 1,
+    icon: "function.svg",
+    paletteLabel: "x",
+    label: function () {
+      return this.name || "x";
```

Comment:
```suggestion
    paletteLabel: "not",
    label: function () {
      return this.name || "not";
```
I think we should replace the names here with the name of the gate, either "not" or "not-gate".